src/api/component.js

```diff
@@ -163,7 +163,7 @@ Component.prototype = Object.create(HTMLElement.prototype, {
       }
 
       if (propertyName) {
-        const propData = data(this, `api/property/${propertyName}`);
```

Comment:
Yep, previously we stored the skate data like this:

```
{
  'api/property/myProperty': { ... },
  'api/property/myOtherProperty': { ... },
  'attributeLinks': { ... },
  'propertyLinks': { ... },
}
```

Now we're storing it like this:

```
{
  'props': {
    'myProperty': { ... },
    'myOtherProperty': { ... },
  },
  'attributeLinks': { ... },
  'propertyLinks': { ... },
}
```